packages/docs/operations-manual/key-management/detailed.md

```diff
@@ -0,0 +1,148 @@
+# Detailed Description of Key Types
+This page provides a detailed description of the various key prototypes found in the Celo protocol, as well as small examples of how to designate key types in the Celo proof-of-stake smart contracts.
+
+## Undesignated
```

Comment:
Celo Account

Any private key generated for use in the protocol has a corresponding Account and Address. The Address is the last 20 bytes of the hash of the public key which corresponds to private key. These keys...